modules/nf-core/propr/propd/main.nf

```diff
@@ -4,21 +4,23 @@ process PROPR_PROPD {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/r-propr:5.0.3':
-        'biocontainers/r-propr:5.0.3' }"
+        'oras://community.wave.seqera.io/library/bioconductor-limma_r-propr:b9f28a713aced4ae' :
```

Comment:
This needs to be the https link (see https://nfcore.slack.com/archives/CJRH30T6V/p1727558952383079)